css/style.css

```diff
@@ -86,92 +107,132 @@ button {
 
 .secondary-heading {
   font-size: 44px;
-  line-height: 56px;
+  line-height: 1.27;
   text-align: center;
 }
 
 /********************* Header styles *****************************/
 
 header {
   margin: 30px 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
 }
 
 .logo-block {
+  display: flex;
+  align-items: center;
   font-size: 32px;
-  line-height: 42px;
+  line-height: 1.31;
   gap: 16px;
 }
 
-.logo-block img {
-  width: 36px;
-  height: 40px;
-}
-
 /************* Header navigation styles *******************/
 
 .nav-menu-items {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: stretch;
+  width: 100%;
   gap: 32px;
 }
 
-.nav-menu-item a {
+.nav-menu-item {
   font-size: 16px;
-  line-height: 24px;
+  line-height: 1.5;
   letter-spacing: 10%;
 }
 
-.dropdown img {
-  width: 15px;
-  height: 15px;
+/******** Needed to finish popup menu!!!  ***************/
+
+.nav-menu-item:hover {
+  border-bottom: 1px solid var(--btn-primary-color);
+}
+
+.nav-dropdown-item {
+  position: relative;
+  /* display: inline-block; */
+}
+
+.arrow-down {
+  vertical-align: middle;
+  margin-left: 4px;
+  padding-bottom: 4px;
+}
+
+.dropdown-content {
+  display: none;
+  width: 221px;
+  position: absolute;
+  left: 0;
+  top: 100%;
+  margin-top: 18px;
+  padding: 16px 20px;
+  background-color: var(--secondary-color);
+  box-shadow: 0 10px 25px rgba(71, 92, 133, 0.1);
+  z-index: 1;
+}
+
+.dropdown-content::before {
+  content: "";
+  position: absolute;
+  top: -20px;
+  left: 12px;
+  border: 10px solid transparent;
+  border-bottom-color: var(--secondary-color);
+}
+
+.nav-dropdown-item:hover .dropdown-content {
+  display: block;
 }
 
-.dropdown-content a {
+.dropdown-item {
   font-size: 16px;
-  line-height: 24px;
+  line-height: 1.5;
   letter-spacing: 10%;
+  margin-bottom: 14px;
+}
+
+.dropdown-item-link:hover {
+  color: var(--btn-primary-color);
 }
 
 .btn-box {
+  display: flex;
+  align-items: center;
   gap: 20px;
 }
 
-.switch-mode {
+.darkmode-toggle + .switch-mode::before {
+  display: inline-block;
+  vertical-align: middle;
+  text-align: center;
+  line-height: 60px;
+  content: url(../img/icons/switch.svg);
```

Comment:
Use quotes for url():
`url("../img/icons/switch.svg")`

Reply:
#fixed